data/plugins/language_python.lua

```diff
@@ -6,8 +6,12 @@ syntax.add {
   files = { "%.py$", "%.pyw$", "%.rpy$" },
   headers = "^#!.*[ /]python",
   comment = "#",
+  block_comment = '"""',
   patterns = {
     { pattern = { "#", "\n" },                 type = "comment"  },
+    { regex   = { '^\\s*"""', '"""' },         type = "comment"  },
```

Comment:
On `master` you can do it with patterns:
```suggestion
    { pattern = { '^%s*"""', '"""' },         type = "comment"  },
```